rules/falco_rules.yaml

```diff
@@ -1677,6 +1677,10 @@
 - list: trusted_images
   items: []
 
+# Backport compatibility
```

Comment:
Can you add a longer comment block like the one for trusted_images that specifically says this is only for backwards compatibility and instead the changes should be made to *_privileged_images or _sensitive_mount_images if possible, and *_privileged_containers/_sensitive_mount_containers as a backup?